packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -48,18 +36,27 @@ export type DestinationOptions = DestinationAccessorOptions &
  *
  * If you want to get a destination only from a specific source, use the corresponding function directly
  *  (`getDestinationFromEnvByName`, `destinationForServiceBinding`, `getDestinationFromDestinationService`).
- * @param name - The name of the destination to be retrieved.
- * @param options - Configuration for how to retrieve destinations from the destination service.
+ * @param destinationFetchOptions - The DestinationFetchOptions to retrieve the destination.
  * @returns A promise returning the requested destination on success.
  * @internal
  */
 export async function getDestination(
-  name: string,
-  options: DestinationOptions = {}
+  destinationFetchOptions: DestinationFetchOptions
 ): Promise<Destination | null> {
+  destinationFetchOptions = destinationFetchOptions.jwt
+    ? { userJwt: destinationFetchOptions.jwt, ...destinationFetchOptions }
```

Comment:
I think we can rename the `userJwt` to `jwt` and get rid of this spread.

Reply:
Agreed